cmd/convox/services.go

```diff
@@ -66,66 +67,67 @@ func init() {
 	}
 
 	stdcli.RegisterCommand(cli.Command{
-		Name:        "services",
-		Description: "manage services",
+		Name:        "resources",
+		Aliases: []string{"services"},
+		Description: "manage external resources [prev. services]",
 		Usage:       "",
 		Action:      cmdServices,
 		Flags:       []cli.Flag{rackFlag},
 		Subcommands: []cli.Command{
 			{
 				Name:            "create",
-				Description:     "create a new service.",
+				Description:     "create a new resource.",
```

Comment:
Created a [Trello card](https://trello.com/c/4MutugIf/1204-standardize-use-of-periods-in-cli-usage-strings).